backend/src/api/model/block/mutations.rs

```diff
@@ -346,6 +400,11 @@ pub(crate) struct NewVideoBlock {
 }
 
 
+#[derive(GraphQLInputObject)]
+pub(crate) struct UpdateTitleBlock {
+    content: Option<String>,
```

Comment:
Why is this optional? Title blocks without content don't really make sense to me? And since there is nothing else to update...

Reply:
Sure, but there could be something else in the future, in which case this should be optional to allow for partial updates. I can change it if you insist, but I don't really see a need. This way it's consistent with all the other update blocks, too. :shrug: